frontend/packages/ceph-storage-plugin/src/components/independent-mode/utils.ts

```diff
@@ -25,9 +25,8 @@ export const checkError = (data: DataState): ErrorType[] => {
 };
 
 export const checkForIndependentSupport = (csv: ClusterServiceVersionKind): boolean => {
```

Comment:
why do we still need this function?

Reply:
We need this to enable some pieces of independent installation.